tests/magics/tcompilesetting.nim

```diff
@@ -0,0 +1,19 @@
+discard """
+cmd: "nim c --nimcache:myNimCache --nimblePath:myNimblePath $file"
```

Comment:
this will make `git status` not clean after running tests, eg:
`nim c -r testament/testament r tests/magics/tcompilesetting.nim`
git status will show `myNimCache/` as untracked file;

maybe add `myNimCache/` to .gitignore, or, better, just use a different compiler option that doesn't have such side effect